ui.apps/src/main/javascript/react-components/copy-translation.js

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+const fs = require('fs');
+const path = require('path');
+
+const languageFolder = './node_modules/@adobe/aem-core-cif-react-components/i18n/en';
+const clientlibFolder = '../../content/jcr_root/apps/core/cif/clientlibs/react-components/resources/lang/en-US';
+
+fs.mkdirSync(clientlibFolder, { recursive: true });
```

Comment:
Shouldn't we rather do that copy in some npm script in `package.json` or maybe better, in the POM of the `ui.apps` project? The `maven-resources-plugin` has a `copy-resources` goal, so we would avoid that the react-components project has to know "where" it is consumed.

Reply:
Done 👍